apps/server/src/modules/learnroom/service/course-export.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain';
+import { LessonService } from '@src/modules/lesson/service';
+import { CourseService } from './course.service';
+import { ImsccFileBuilder } from '../imscc';
+
+@Injectable()
+export class CourseExportService {
```

Comment:
Since this supports IMSCC only we should name it accordingly: CourseExportCommonCartridgeService or CourceExportIMSCCService

Reply:
Done.